guides/v3.6.0/configuring-ember/debugging.md

```diff
@@ -19,29 +19,18 @@ export default Application.extend({
   LOG_TRANSITIONS_INTERNAL: true
 });
 ```
-## Views / Templates
-
-#### Log view lookups
-
-```javascript {data-filename=config/environment.js}
-ENV.APP.LOG_VIEW_LOOKUPS = true;
-```
-#### View all registered templates
-```javascript
-Ember.keys(Ember.TEMPLATES)
```

Comment:
```
$ Ember.keys(Ember.TEMPLATES)
TypeError: Ember.keys is not a function[Learn More]
```

Reply:
> I don't think Ember.TEMPLATES is populated anymore.

Confirmed in latest canary, `Ember.TEMPLATES` is always an empty object